backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -706,35 +705,10 @@ private async void OnIsarStartup(object? sender, MqttReceivedArgs mqttArgs)
             }
 
             _logger.LogInformation(
-                "Received ISAR restart event for robot {robotName} with ISAR id {isarId}. Will restart ongoing missionRun, if any, by moving it back to the queue.",
+                "Received ISAR restart event for robot {robotName} with ISAR id {isarId}.",
```

Comment:
The entire startUpMessage can probebly removed as it doesn't do anything anymore. Suggest making a new issue to handle this